### DISCUSSION TITLE
Hide "Add" if option already exists?

### DISCUSSION THREAD

**Discussion by seabasss** [orchidjs/tom-select#189] [Ideas]
Is there a way to hide the "Add..." row if you type something that already exists?
Maybe there is a way to use `createFilter` to compare with existing options and return false?
Probably lowercase-compare on both sides.

Thank you!

**Comment by oyejorge** [comment]
You could return null from the [```option_create``` template](https://tom-select.js.org/docs/#render-templates).

**Comment by seabasss** [comment]
Thanks! But what’s the actual comparison part to only hide ”Add” if it already exists? Just return null would always hide ”Add” right?

Edit. I think I figured it out:

```
option_create: function(data, escape) {
    if(this.items.includes(data.input.toLowerCase())) {
        return null;
    } else {
        // Add
    }
}
```

Thank you!

**Comment by seabasss** [comment]
I'm back with this question again. `this.items` worked when I had static options, but that is an empty array when loading from a remote source. What can I use to return null comparing to remote results?

I'm looking into something like this now but it's getting pretty ugly `this.dropdown_content.querySelectorAll...`

Ideas?

**Reply by oyejorge** [reply]
If you want to check for existing options, you'll want to use ```this.options``` instead of ```this.items```

**Reply by seabasss** [reply]
Awesome! That did it! Thanks
